TemplePlus/condition.cpp

```diff
@@ -5352,6 +5375,44 @@ int ItemCallbacks::BucklerToHitPenalty(DispatcherCallbackArgs args)
 	return 0;
 }
 
+int __cdecl ItemCallbacks::MaxDexBonus(objHndl armor)
+{
+	auto res = itemCallbacks.oldMaxDexBonus(armor);
+	
+	//Query for max dex bonus adjustment
+	if (armor) {
+		auto parent = inventory.GetParent(armor);
+		if (parent) {
+			auto obj = objSystem->GetObject(parent);
+			if ((obj != nullptr) && (obj->GetInt32(obj_f_type) == obj_t_pc)) {
```

Comment:
Why PC only? How about NPC?

Reply:
Why indeed.
Also please use obj methods like IsPc, IsNpc, IsCritter in case we ever refractor this.

Reply:
Both valid suggestions. I'll update this in my next pull request.